modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Fix line separator scrolling in floating toolbars. #TINY-8948
```

Comment:
Maybe this instead not sure that we use the `Fix` prefix on other changelog items. But it seems that we have no rules since some times it's `Fixed bla bla` some times it's just `Bla bla` without the fixed prefix.

```suggestion
- Line separator scrolling in floating toolbars. #TINY-8948
```